ci/qa/security-tests

```diff
@@ -4,4 +4,4 @@ cd $(dirname $0)/../../
 
 "wget -q https://github.com/fabpot/local-php-security-checker/releases/download/v1.0.0/local-php-security-checker_1.0.0_linux_amd64 -O local-php-security-checker && chmod +x ./local-php-security-checker && ./local-php-security-checker",
 source "/usr/local/nvm/nvm.sh"
-yarn audit
\ No newline at end of file
+yarn audit
```

Comment:
There she is (the linefeed) ;)

In a perfect world you would have added this change to the previous commit. No worries, leave it here for the time being!